### DISCUSSION TITLE
How can customize tab width of code blocks ?

### DISCUSSION THREAD

**Discussion by ghost** [willfaught/paige#89] [Q&A] [answered]
In the text editor, I have set tab size to 4. But when I view the code block in the browser, the tab-width is 8 characters.

How can I ensure that tab is 4 characters in the browser ?

**Comment by willfaught** [comment, marked as answer]
This is a Hugo issue. See https://discourse.gohugo.io/t/possible-to-specify-tabwidth-using-highlight-shortcode/49939.
